src/singletons/Settings.hpp

```diff
@@ -524,6 +524,8 @@ class Settings
         HelixTimegateOverride::Timegate,
     };
 
+    BoolSetting enableHelixChatSend = {"/misc/helixChatSend", true};
```

Comment:
I would prefer if this is an enum setting with the values:
 - Default (this should be the default value)
 - IRC
 - Helix